envcov.sh

```diff
@@ -10,7 +10,7 @@ ENV_DIR="${DIR}"
 if [[ $DIR == *Pods/XcodeCoverage* ]]
```

Comment:
I think the `[[ ... ]]` construct is bash-specific, so maybe add `#!/bin/bash` or some such at the top?